.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
   "extends": [
-    "ts-important-stuff",
+    "ts-react-important-stuff",
     "plugin:prettier/recommended"
   ],
+  "plugins": [
+    "eslint-plugin-react-hooks"
+  ],
```

Comment:
This is (1) incorrect and (2) unnecessary. It is incorrect because it should be `"plugins": ["react-hooks"]` (as specified [in the react docs](https://reactjs.org/docs/hooks-rules.html#eslint-plugin)). And it's unnecessary because [it is already turned on inside of eslint-config-react-important-stuff](https://github.com/joeldenning/eslint-config-react-important-stuff/blob/6404b3ec64378528d7fe3999e5df68766f08e44f/eslint-config-react-important-stuff.js#L8), which is extended by eslint-config-ts-react-important-stuff.

If for some reason ts-react-important-stuff isn't turning on the react-hooks plugin, we should investigate and fix that.